hack/conformance.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# This script should be executed through the makefile via `make conformance`.
+
+set -euo pipefail
+
+echo 'Preparing directory for artifacts'
+export ARTIFACTS=_out/artifacts/conformance
+mkdir -p ${ARTIFACTS}
+
+echo 'Obtaining KUBECONFIG of the development cluster'
+export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
+
+echo 'Executing conformance tests and wait for them to finish'
+sonobuoy run --wait --plugin _out/manifests/release/conformance.yaml
+
+trap "{ echo 'Cleaning up after the test execution'; sonobuoy delete --wait; }" EXIT SIGINT SIGTERM
+
+echo 'Downloading report about the test execution'
+results_archive=${ARTIFACTS}/$(cd ${ARTIFACTS} && sonobuoy retrieve)
+
+echo 'Results:'
+sonobuoy results ${results_archive}
+
+echo "Extracting the full report and keep it under ${ARTIFACTS}"
+tar xf ${results_archive} -C ${ARTIFACTS}
+cp ${ARTIFACTS}/plugins/kubevirt-conformance/results/global/junit.xml ${ARTIFACTS}/junit.conformance.xml
+
+echo 'Evaluating success of the test run'
+sonobuoy results ${results_archive} | grep -q 'Status: passed' || {
+    echo 'Conformance suite has failed'
+    exit 1
+}
```

Comment:
maybe worth to add a "success" otherwise?
or just retcode 0 and not errors printed would be enough

Reply:
Done